GitSearch/Models/SpinnerViewController.swift

```diff
@@ -10,6 +10,17 @@ import UIKit
 
 class SpinnerViewController: UIViewController {
     var spinner = UIActivityIndicatorView(style: UIActivityIndicatorView.Style.gray)
+    var owner: UIViewController
```

Comment:
This creates [retain cycle](https://docs.swift.org/swift-book/LanguageGuide/AutomaticReferenceCounting.html) after spinner view controller gets added to owner (it will be retained as a child).
Also, `UIViewController` already has [`parent`](https://developer.apple.com/documentation/uikit/uiviewcontroller/1621362-parent) property.